framework/controller/loader.go

```diff
@@ -278,7 +280,7 @@ func (l *loader) loadActionParams(params []*parser.Param) (inputs []*ActionParam
 		inputs = append(inputs, l.loadActionParam(param, nth, numParams))
 	}
 	if len(inputs) > 0 {
-		l.imports.Add("github.com/livebud/bud/runtime/controller/request")
+		l.imports.Add("github.com/livebud/bud/framework/controller/controllerrt/request")
```

Comment:
All helper runtimes live within the generator's package inside the framework directory.